includes/class-activation.php

```diff
@@ -0,0 +1,123 @@
+<?php
+/**
+ * Activation handler
+ *
+ * @package AffiliateWP\ActivationHandler
+ * @since   1.0.0
```

Comment:
The file is new and introduced in 1.0.3
```suggestion
 * @since   1.0.3
```